### PR TITLE
docs: Address comments from admin partitions docs changes around networking requirements

### PR DESCRIPTION
**Pull request by david-yu**
### Description
Describe why you're making this change, in plain English.

### Testing & Reproduction steps
* In the case of bugs, describe how to replicate
* If any manual tests were done, document the steps and the conditions to replicate
* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding

### Links
Include any links here that might be helpful for people reviewing your PR (Tickets, GH issues, API docs, external benchmarks, tools docs, etc). If there are none, feel free to delete this section.

Please be mindful not to leak any customer or confidential information. HashiCorp employees may want to use our internal URL shortener to obfuscate links.

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] not a security concern
* [ ] checklist [folder](./../docs/config) consulted

### CODE REVIEW

File: website/content/docs/enterprise/admin-partitions.mdx [hunk 1]
```diff
@@ -70,13 +70,13 @@ Your Consul configuration must meet the following requirements to use admin part
 
 ### General Networking Requirements
 
-All Consul clients must be able to communicate with the Consul servers in the `default` partition. All servers must also be able to communicate with the clients.
+All Consul clients must be able to communicate with the Consul servers for the assigned partition that is assigned to. All servers must also be able to communicate with the clients.
```
Comment (blake) [inline website/content/docs/enterprise/admin-partitions.mdx:73]:
```suggestion
All Consul clients must be able to initiate Gossip, HTTP, and RPC connections to the servers. All servers must also be able to initiate Gossip connections to the clients.
```

**Comment by david-yu** [comment]
@Blake All good to go here? Should it say HTTPS instead of HTTP?


`All Consul clients must be able to initiate Gossip, HTTPS and RPC connections to the servers. All servers must also be able to initiate Gossip connections to the clients.`

**Review by blake — APPROVED** [review]
@david-yu Changing `HTTP` to `HTTPS` should be all that's needed. Thanks!

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/673566.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 499fbdabaf26d43281d7f4a8f1fe8bdbad2b262d onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 499fbdabaf26d43281d7f4a8f1fe8bdbad2b262d onto `release/1.12.x` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 499fbdabaf26d43281d7f4a8f1fe8bdbad2b262d onto `release/1.11.x` succeeded!

**Pull request merged**